client/scraprpi/scraprpi.go

```diff
@@ -14,89 +13,115 @@ import (
 	"github.com/mackerelio/go-osstat/memory"
 )
 
-func CheckError(err error) {
+func CheckError(context string, err error) {
 	if err != nil {
-		panic(err)
+		fmt.Printf("Error: %s\n", context)
+		log.Fatalln(err)
 	}
 }
 
 type SystemInfo struct {
-	HardwareID  string  `json:"HardwareID"`
+	MacAddress  string  `json:"MacAddress:"`
 	CPUuserLoad float64 `json:"CPUuserLoad"`
-	CPUidle     float64 `json:"CPUidle"`
 	TotalMemory int64   `json:"TotalMemory"`
```

Comment:
this struct also needs to be split into two structs, one for `info` other for `metric`